src/main/java/XlsxExcelFile.java

```diff
@@ -81,8 +82,28 @@ public void writeCell(String content, int rowIdx, int columnIdx) {
             Logger.printError("Cell at column " + columnIdx + " does yet not exist. Creating new cell...");
             cell = row.createCell(columnIdx);
         }
-
-        cell.setCellValue(content);
+        
+		cell.setCellType(content.type);
+        switch(content.type) {
```

Comment:
It would be cool to export this logic to the class `CellContent`.
I am thinking of something like:

```java
// getValue() is defined in CellContent
cell.setCellValue(content.getValue());
``